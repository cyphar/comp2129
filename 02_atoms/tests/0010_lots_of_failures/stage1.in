STAT
STAT 2 3
STAT a
STAT    
START 2 3 1
STAT
STAT 2 3
STAT a
STAT  	
