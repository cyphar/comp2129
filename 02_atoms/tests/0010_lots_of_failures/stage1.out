Game Not In Progress

Game Not In Progress

Game Not In Progress

Game Not In Progress

Game Ready
Red's Turn

Player Red:
Grid Count: 0

Player Green:
Grid Count: 0

Invalid Command

Invalid Command

Player Red:
Grid Count: 0

Player Green:
Grid Count: 0

